.github/ISSUE_TEMPLATE.md

```diff
@@ -6,14 +6,17 @@ A summary of the issue.
 
 ### Steps to Reproduce
 
-1. This is the first step
-2. This is the second step, etc.
+1. (for example) Start a new project with `wagtail start myproject`
+2. Edit models.py as follows...
```

Comment:
We should provide further examples (worded as questions?) to direct the answers. For example,

```
4. What did you change in the settings?
5. What content did you create in the admin?
```